server/user_account/views.py

```diff
@@ -161,3 +160,23 @@ def post(self, request):
         qs = CustomUser.objects.filter(first_name=first_name)
         serializer = ClientGridSerializer(instance=qs, many=True)
         return Response(serializer.data, status=status.HTTP_200_OK)
+
+
+class ModifyClient(generics.GenericAPIView):
+    http_method_names = ['post']
+    queryset = CustomUser.objects.all()
+    serializer_class = ClientGridSerializer
+    permission_classes = [IsAuthenticated]
+
+    def post(self, request):
+        data = request.data
+        variable_column = data.get('category', '')
+        entry = data.get('field', '')
+        id1 = data.get('id', '')
+        if entry == 'false':
+            entry = False
+        elif entry == 'true':
+            entry = True
+        CustomUser.objects.filter(id=id1).update(**{variable_column: entry})
+        serializer = ClientGridSerializer(instance=self.get_queryset(), many=True)
+        return Response(serializer.data, status=status.HTTP_200_OK)
```

Comment:
For the moment this method is okay by you can simplify all of what is inside the post method by using a viewset. If you want you can checkout what i did in my branch (manage_organization_backend) which is now a pull request. And if you decide to convert it to a viewset you can just combine it with the other view you created class AccessAllClients(viewsets.ModelViewSet) that will have both methods under the hood. If you need I can explain on a call more about the power fo viewsets, let me know.

Reply:
Also, on a side-note, this method isn't tested.